crates/shared_types/src/directive.rs

```diff
@@ -464,6 +532,14 @@ mod tests {
             })
         );
 
+        assert_eq!(
```

Comment:
can we add a test with multiple string_list given?

I see that it should work, but I think we should test.

Reply:
Added a test with two entries:

```rust
        assert_eq!(
            parse_to_directive("attr.string_list: plugins -> { //x/y/z:artifact, bar }"),
....
```